projects/media-viewer/src/lib/store/actions/router.action.spec.ts

```diff
@@ -0,0 +1,46 @@
+import * as fromRouter from './router.action';
```

Comment:
as above, not sure we need these now. and we can always add them when needed

Reply:
removed

Reply:
removed